### PR TITLE
Added signin.html

### PR DESCRIPTION
**Pull request by KayviaHarriott**
Added signin.html along with styling in signin.css

### CODE REVIEW

File: src/signin.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+body {
+    font-family: "Montserrat", sans-serif;
+  }
+  
+  .form-group {
```
Comment (nbkhope) [inline src/signin.css:5]: Looks like we have the same class names and styles as sign up. How about we reuse them?

Basically, import the stylesheet from signup  into the signin, in addition to this signin.css

```
<link href="signup.css">
<link ... href="signin.css">
```

That way you dont repeat yourself. If any style is different, you can keep them in this separate signin.css file.

Comment (KayviaHarriott) [reply src/signin.css:5]: Yes, you're right! Will do so in the next code changes/pull request

Comment (nbkhope) [reply src/signin.css:5]: @KayviaHarriott you dont have to create a new pull request. All you have to do is add new commits to the same branch `KayviaHarriott:feature/add-sign-in-page`  and this pull request will be updated automatically.

Comment (KayviaHarriott) [reply src/signin.css]: Got it! Thanks, I've made a new commit on the same branch

**Review by nbkhope — APPROVED** [review]

**Pull request merged**